Formula/lv2.rb

```diff
@@ -14,8 +13,11 @@ class Lv2 < Formula
   end
 
   def install
-    system "./waf", "configure", "--prefix=#{prefix}", "--no-plugins"
+    system "./waf", "configure", "--prefix=#{prefix}", "--lv2dir=#{lib}/lv2/"
     system "./waf", "build"
     system "./waf", "install"
   end
+
+  test do
```

Comment:
it's not only documentation, it's a C API. Think all the header files, but no implementation files.